.github/workflows/linux_builds.yml

```diff
@@ -82,10 +82,8 @@ jobs:
           sudo cp -f misc/ci/sources.list /etc/apt/sources.list
           sudo apt-get update
           # The actual dependencies
-          sudo apt-get install build-essential pkg-config libx11-dev libxcursor-dev \
-              libxinerama-dev libgl1-mesa-dev libglu-dev libasound2-dev libpulse-dev \
-              libdbus-1-dev libudev-dev libxi-dev libxrandr-dev yasm xvfb wget unzip \
-              llvm libspeechd-dev speech-dispatcher fontconfig libfontconfig-dev libxkbcommon-dev
+          sudo apt-get install build-essential pkg-config libgl1-mesa-dev libglu-dev \
+              xvfb wget unzip llvm speech-dispatcher fontconfig
```

Comment:
Do we still need speech-dispatcher and fontconfig libraries installed?

Reply:
Probably not, we do not have test for `SystemFont` and there's not much to automatically test for TTS.